### PR TITLE
Strata/Platform/Host - UI: minor updates

### PR DESCRIPTION
**Pull request by deetz99**
*Issue:*

- bcgov/entity/issues/24807
- bcgov/entity/issues/24944
- bcgov/entity/issues/24897

*Description of changes:*
- fix strata application payload
- log user out and redirect to housing page when declining tos
- fix modal close button not working

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: strr-base-web/app/composables/useStrrModals.ts [hunk 1]
```diff
@@ -8,19 +8,21 @@ import {
 
 export const useStrrModals = () => {
   const modal = useModal()
-  const { t } = useI18n()
-  const connectNav = useConnectNav()
+  const t = useNuxtApp().$i18n.t
```
Comment (deetz99) [inline strr-base-web/app/composables/useStrrModals.ts:11]: not sure why the useI18n composable is causing problems suddenly

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-394-gcxeaqr3.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-strata-hotel-dev--pr-394-fsh76qnt.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-hosts-dev--pr-394-vfbzccb2.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-394-gcxeaqr3.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-strata-hotel-dev--pr-394-fsh76qnt.web.app

File: strr-strata-web/package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "strr-strata-web",
   "private": true,
   "type": "module",
-  "version": "0.0.17",
+  "version": "0.0.19",
```
Comment (deetz99) [inline strr-strata-web/package.json:5]: other pr was 0.0.18

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-394-gcxeaqr3.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-strata-hotel-dev--pr-394-fsh76qnt.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-hosts-dev--pr-394-vfbzccb2.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-394-gcxeaqr3.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-strata-hotel-dev--pr-394-fsh76qnt.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-hosts-dev--pr-394-vfbzccb2.web.app

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**